Library/Homebrew/utils/github.rb

```diff
@@ -562,6 +562,7 @@ def self.fetch_pull_requests(name, tap_remote_repo, state: nil, version: nil)
 
     pull_requests = []
     API.paginate_graphql(graphql_query, variables:) do |result|
+      return [] if result.nil?
```

Comment:
Can this be moved to the beginning of the `fetch_pull_requests` method instead? Thanks!

Reply:
Changed it to check the env var directly, let me know if this is what you had in mind.